### PR TITLE
chore(release): 3.21.1

### PR DESCRIPTION
**Pull request by mcous**
## Overview

This PR prepares the 3.21.1 hotfix branch for release now that the alpha build has passed QA

### CODE REVIEW

**Review by sfoster1 — APPROVED** [review]
Good with me!

**Review by nusrat813 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [3.21.1-alpha.0](https://github.com/Opentrons/opentrons/compare/v3.21.0...v3.21.1-alpha.0) (2020-10-09)
+## [3.21.1](https://github.com/Opentrons/opentrons/compare/v3.21.0...v3.21.1) (2020-10-13)
```
Comment (bndo) [inline CHANGELOG.md:6]: Should this and others be `2020-10-14`?

Comment (mcous) [reply CHANGELOG.md:6]: PR was opened on the 13th so... ¯\\\_(ツ)\_/¯

**Pull request merged**